opentracing-reactive-grpc/src/test/java/io/quarkus/qe/AbstractPingPongResourceIT.java

```diff
@@ -0,0 +1,81 @@
+package io.quarkus.qe;
+
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.HttpStatus;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.qe.containers.JaegerTestResource;
+import io.quarkus.test.bootstrap.JaegerService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.services.JaegerContainer;
+import io.quarkus.test.services.QuarkusApplication;
+
+public abstract class AbstractPingPongResourceIT {
+
+    private static final String PING_ENDPOINT = "/%s-ping";
+    private static final String PONG_ENDPOINT = "/%s-pong";
+
+    private static final String PING_RESOURCE = "PingResource";
+    private static final String PONG_RESOURCE = "PongResource";
+
+    private static final int TRACE_PORT = 14268;
+    private static final int REST_PORT = 16686;
+
+    @ConfigProperty(name = JaegerTestResource.JAEGER_API_ENDPOINT, defaultValue = "http://localhost:16686/api/traces")
+    String jaegerEndpoint;
+
+    @JaegerContainer(image = "jaegertracing/all-in-one:latest", tracePort = TRACE_PORT, restPort = REST_PORT, expectedLog = "\"Health Check state change\",\"status\":\"ready\"")
```

Comment:
You don't need to specify the image, tracePort, restPort and expectedLog, using simply `@JaegerContainer` works fine: if using ` .withProperty("quarkus.jaeger.endpoint", jaeger::getRestUrl);` and `.when().get(jaeger.getTraceUrl() + "?traceID=" + pingTraceId)`